BookStore.EF6/Services/BookReader.cs

```diff
@@ -9,9 +9,11 @@ namespace BookStore.EF6.Services
 {
     public class BookReader : IBookReader
     {
+        IQueryable<EBook> EBooks => new StoreContext().Books.Include("Authors").AsNoTracking();
```

Comment:
@ma-gu @johnnyoshika  I would put high level abstraction + public interface stuff to the top, so we can read from "what" to "how" while going down through the file. It means having implementation at the end of the file in the order of dependencies. Let's move EBooks at the very end as it does not depend on anything and not public.